lib/scala/searcher/src/main/scala/org/enso/searcher/sql/SqlSuggestionsRepo.scala

```diff
@@ -149,11 +161,11 @@ final class SqlSuggestionsRepo(val db: SqlDatabase)(implicit
       Seq[(Long, Int, String)]
     )
   ] =
-    db.run(renameProjectQuery(oldName, newName))
+    db.run(renameProjectQuery(oldName, newName).transactionally)
 
   /** @inheritdoc */
   override def currentVersion: Future[Long] =
-    db.run(currentVersionQuery)
+    db.run(currentVersionQuery.transactionally)
 
```

Comment:
Why is `currentVersion` in a transaction? It seems like it should perform a single query, so I don't see the reason of wrapping it into a transaction, unless I'm missing something?

I could not comment there, but why is `insertBatch` _not_ a transaction? It seems like actually this operation would benefit from it as it inserts multiple records.

Reply:
I had to re-check everything once again. You're right here, transaction would benefit only when a bunch of records is inserted at once, because in SQLite each insert statement is executed in its own transaction, and wrapping the batch query in the begin-end (transaction) statement makes them run in a single transaction. 
In the end, I left transactions only in a few places when there are multiple records are updated.

`insertBatch` was an internal method, and I skipped it when checking public methods or the interface. You're right it'll also benefit from wrapping it in the transaction